pkg/listener/new_result.go

```diff
@@ -88,34 +91,8 @@ func (l *ResultListener) Listen(event report.LifecycleEvent) {
 		return
```

Comment:
why has this listeners so many changes when we agreed to not touch this logic for now?